src/amo/components/LandingPage.scss

```diff
@@ -1,5 +1,58 @@
 @import "~amo/css/inc/vars";
+@import "~core/css/inc/mixins";
 
 .LandingPage {
   padding: $padding-page;
 }
+
+.LandingPage-header {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  margin: $padding-page $padding-page ($padding-page * 2);
+}
+
+.LandingPage-header-top {
+  align-items: center;
+  display: flex;
+  justify-content: center;
+  margin-bottom: $padding-page * 2;
+}
+
+.LandingPage-header-text {
+  @include margin-start(20px);
+
+  display: inline-block;
+  max-width: 350px;
```

Comment:
Here it is at 800x600. I just made up the number for something that seemed like a reasonable width to wrap at.

![screen shot 2017-03-22 at 18 26 35](https://cloud.githubusercontent.com/assets/211578/24225185/2af6604a-0f2d-11e7-87f0-bf1bb8eb2bd4.png)